### PR TITLE
Dabaste unit tests

### PR DESCRIPTION
**Pull request by ltuffery**
# Description

# Checklist
The following items should be checked before the pull request can be merged.
- [x] Unit tests added (if applicable).
- [x] The code formatted (e.g. with `npm run format`)

### CODE REVIEW

File: api/src/Managers/UserManager.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        ]);
+
+        if (!is_null($user) && password_verify($password, $user->password)) {
+            $this::$user = $user;
+
+            return true;
+        }
+
+        return false;
+    }
```
Comment (ltuffery) [inline api/src/Managers/UserManager.php:45]: Maybe don't make a user manager but just put the functions in the User model.

File: api/src/Model/Model.php [hunk 2]
```diff
@@ -162,12 +166,37 @@ public static function all(): array
...
 
+    public function getData(): array
+    {
+        $class = new ReflectionClass($this);
+        $data = [];
+
+        foreach ($class->getProperties(ReflectionProperty::IS_PUBLIC) as $property) {
+            if (!isset($this->{$property->getName()}) || $property->getName() == 'id') {
+                continue;
+            }
```
Comment (ltuffery) [inline api/src/Model/Model.php:187]: Leave the `id` in the data

File: api/src/Testing/Cases/DatabaseTestCase.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            return $sql;
+        }
+
+        $lines = explode("\n", $sql);
+        $alterSqlLine = $lines[0];
+        $lines = array_slice($lines, 1);
+
+        foreach ($lines as $key => $value) {
+            if (!empty($value)) {
+                $lines[$key] = $alterSqlLine . ' ' . str_replace("ADD", "ADD COLUMN", $value);
```
Comment (ltuffery) [inline api/src/Testing/Cases/DatabaseTestCase.php:54]:
```suggestion
                $lines[$key] = $alterSqlLine . str_replace("ADD", " ADD COLUMN", $value);
```

File: api/src/config.php [hunk 4]
```diff
@@ -12,18 +13,22 @@
     getenv('MYSQL_PASSWORD'),
 ]);
 
-Flight::map('user', function () {
+Flight::map('user', function() {
```
Comment (ltuffery) [inline api/src/config.php:16]:
```suggestion
Flight::map('user', function () {
```

**Review by swotex — APPROVED** [review]
LGFM

**Pull request merged**